pkg/options/options.go

```diff
@@ -190,5 +194,10 @@ func (o *Options) Validate() error {
 			return fmt.Errorf("resource %s can't be sharded by field selector spec.nodeName", x)
 		}
 	}
+
+	if o.AutoGoMemlimitRatio <= 0.0 || o.AutoGoMemlimitRatio > 1.0 {
+		return fmt.Errorf("value for  --auto-gomemlimit-ratio=%q must be greater than 0 and less than or equal to 1", o.AutoGoMemlimitRatio)
```

Comment:
```suggestion
		return fmt.Errorf("value for --auto-gomemlimit-ratio=%q must be greater than 0 and less than or equal to 1", o.AutoGoMemlimitRatio)
```

looks like I added one whitespace too much here.

Reply:
Might have been my copy-n-paste. Fixed.